.github/workflows/stale.yml

```diff
@@ -1,14 +1,14 @@
-name: 'Mark and close stale issues'
+name: "Mark and close stale issues"
```

Comment:
Just curious if there is any specific reason to switch single quotes to double quotes. Escape character in the string?

Reply:
It's formatted by [Prettier][1] plugin of vscode when mergeing and resolving conflicts. I just keep the formatting result because it do not change the content.

[1]: https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode